Create Input Breadboard subsystem.md

K-Robbins-4687 opened the pull request.

K-Robbins-4687 committed:
Create Input Breadboard subsystem.md

K-Robbins-4687 committed:
Create readme

K-Robbins-4687 committed:
Add files via upload

K-Robbins-4687 committed:
Update Input Breadboard subsystem.md

K-Robbins-4687 committed:
Update 2 Input Breadboard subsystem.md

K-Robbins-4687 committed:
Update Input Breadboard subsystem.md

1) Addressed the level of interactivity
2) Used an auto variable Ohm-meter (see reference 2)
3) Changed the sensor circuit and blocks accordingly

JesseTNRoberts reviewed (commented):
I don't understand how the series connector will permit arbitrary series configurations. For instance, if I want to put 2 resistors in but not 3 or 4, how could I do that?

K-Robbins-4687 committed:
Update Input Breadboard subsystem.md

1) Addressed the interactivity once more. Now, the only instruction given to the user is to change connections outside the sensor circuit. Now, they can plug any combination of 3 resistors, 3 capacitors, etc. They will be able to make series and parallel connections themselves.

K-Robbins-4687 committed:
Update Input Breadboard subsystem.md

Fixed a small typo

JesseTNRoberts reviewed (commented):
What does this mean that it will be separate? It needs to be part of the car as I understand.

K-Robbins-4687 commented:
The team and I discussed this at length, so I'll tell you what we think. Since the car will be moving as part of the demonstration, we thought it would be best to separate the housing for the input breadboard and the car, so that the user wouldn't have to make changes to the "circuit" of a moving car.

JesseTNRoberts commented:
That's not your call to make and it's not what the customer asked for. They can pick the car up or stop it to adjust the circuit.

K-Robbins-4687 committed:
Update Input Breadboard subsystem.md

1) Fixed mention of separate housing for subsystem

K-Robbins-4687 committed:
Update Input Breadboard subsystem.md

JesseTNRoberts reviewed (commented):
In person review occurred.

K-Robbins-4687 committed:
Update Input Breadboard subsystem (TEMPORARY).md

K-Robbins-4687 committed:
Update Input Breadboard subsystem.md

Made change to sensor circuit as stated by Mr. Roberts

JesseTNRoberts reviewed (commented):
This looks good! How big do the blocks need to be (precise dimensions)? How do you know the clips will hold well? What happens if the user makes a dead short (is that a problem)?  It doesn't look like you've shown that the method for measuring the resistance is sure to be able to measure the highest permitted resistance (all resistors in series).

K-Robbins-4687 committed:
Update Input Breadboard subsystem.md

(1) Added 3d models for the pieces
(2) Addressed the max resistance scenario
(3) Addressed the short circuit scenario

K-Robbins-4687 committed:
Update Input Breadboard subsystem.md

Addressed the design for the blocks

JesseTNRoberts reviewed (commented):
The Gerber files have no similarity to the proposed PCB in the top of the signoff. Further, it won't work.

micahr234 commented on Documentation/Signoff/Input Breadboard subsystem.md:
```diff
@@ -0,0 +1,120 @@
+![image](https://github.com/abdoulm366/TTU-Capstone--Electrical-Class-Kit/assets/158489186/4d2e6df5-77fa-4086-aec4-3450dbacf9bf)
+
+Figure 4: PCB 3D view
+
```
Where is the PCB for the detection circuit?

micahr234 commented on Documentation/Signoff/Input Breadboard subsystem.md:
```diff
@@ -0,0 +1,120 @@
+
+Lastly, the effect of the diode is the easiest to both understand and calculate. Since the diode is a binary-acting device, its output voltage is either high or zero. Most diodes have a 0.7 voltage drop, and since the Peak Inverse Voltage is far less than even 5 volts, all it will have to do is block current going the other way. The analog in port will then find that the voltage is zero, and appr ...
+
+## B.O.M.
+| Component Type: | Quantity: | Link                      | Price:   |
+|-----------------|-----------|---------------------------|----------|
+| Resistors       | 1         | https://shorturl.at/svDE8 | $ 13.99  |
+| Capacitors      | 1         | https://shorturl.at/ouxBV | $ 14.99  |
+| Diodes          | 1         | https://shorturl.at/fswBG | $ 9.99   |
+| Large Perfboard      | 1         | https://shorturl.at/xzG47 | $ 5.50  |
+| PCBs for blocks    | 20 | N/A | $30.00 |
+| Female Socket  | 120 | https://shorturl.at/cEHP4 | $13.20|
+| Jumper Wire (100 Pcs) | 1 | https://shorturl.at/noQ38 | $29.84|
+| Arduino Mega    | 1         | https://shorturl.at/htuxV | $ 48.40  |
+| Max 4167    | 1         | https://shorturl.at/EQUY7    | $ 6.18 |
+| Resistors (1% tolerance)    | 1         | https://shorturl.at/ajrDO      | $ 9.99  |
+|                 |           | Total:                    | $182.08 |
```
Need to include manufacturer and part number in BOM.

K-Robbins-4687 committed:
[temporary]Update Input Breadboard subsystem.md

K-Robbins-4687 committed:
Update Input Breadboard subsystem.md
